### ISSUE TITLE
Arcade CORE a.*.rom initial setup problem

### ISSUE THREAD

**Issue by MacLe7400** [MiSTer-devel/Main_MiSTer#185]
Hello,

I have a problem with initial Arcade Cores setup.

I'm trying to generate` a.*.rom` files from MAME ROMs.

I have my name ROMs (zip files) copied to `/media/fat/_Arcade/mame`

Update script (`/media/fat/#Scripts/update.sh`) script finishes without errors.

I see that all the required COREs and MRAs were downloaded in previous runs:
MRAs to `/media/fat/_Arcade/` and RBFs to `/media/fat/_Arcade/cores/`
Additional directories `mame/ hbmame/` were also created.

I've tried also resync updater by deleting:
`/media/fat/Scripts/.mister_updater/update.last_successful_run`

Still, no errors and `a.*.rom` files weren't generated.

My setup:
```
MiSTer v200325, OS v200122
MiSTer Updater version 4.0.4
```

Any hints how to force generate to convert mame roms to arcade roms?

Regards,
M

**Comment by sorgelig** [comment]
a.rom is not supported any more. use MRA.

**Comment by MacLe7400** [comment]
> a.rom is not supported any more. use MRA.

I see. Where I should store MAME rom files (zip files) ?

**Comment by blackjer** [comment]
mame zips should be stored in /_Arcade/mame

**Comment by MacLe7400** [comment]
I keep the zips in this directory (/_Arcade/mame) but I cannot run any arcade core. 

After launching proper core my device just hangs. Is there a logs somewhere to check what's is going on?

Facts:
1) I don't have additional SDRAM (yet)
2) other cores (which doesn't require additional SDROMS) works fine (like Genesis).
 

> mame zips should be stored in /_Arcade/mame
> […](#)

**Comment by ctafield** [comment]
Are you launching the core or the MRA file? You need to launch the latter. Pretty sure you need the sdram too.
Have you tried using the RetrodrivenMAME script to set it up instead? That’ll put things in the right folders for you.

**Comment by sorgelig** [comment]
For most arcades SDRAM is not required.

**Comment by MacLe7400** [comment]
> For most arcades SDRAM is not required.

Could you list 1-2 arcades which is 100% sure that they works without SDRAM?

**Comment by sorgelig** [comment]
Arkanoid for example

**Comment by MacLe7400** [comment]
> Are you launching the core or the MRA file? You need to launch the latter. Pretty sure you need the sdram too.
> Have you tried using the RetrodrivenMAME script to set it up instead? That’ll put things in the right folders for you.

Yes. I've tried. Script downloaded all necessary files without errors (including MAME roms).

Still.. no luck. Black screen after launching particular arcade cores (I've tried several different cores).

Any other hints?

**Comment by blackjer** [comment]
Can you attach a screenshot of the folder structure on your SD card?

**Comment by MacLe7400** [comment]
> Can you attach a screenshot of the folder structure on your SD card?
> […](#)

Here is /media/fat structure:

```
/media/fat# find . -type d
./config
./config/inputs
./Scripts
./Scripts/.mister_updater
./Scripts/.mame
./Scripts/.RetroDriven
./Scripts/.RetroDriven/MAME
./Scripts/.RetroDriven/MAME_CPS1
./Scripts/.RetroDriven/HBMAME
./Scripts/.RetroDriven/MRA
./Scripts/.RetroDriven/MRA_CPS1
./linux
./linux/soundfonts
./linux/mt32-rom-data
./mame
./mame/roms
./#Scripts
./RetroDriven_Logs
./RetroDriven_Logs/MAME
./bootrom
./_CPS1
./_CPS1/mame
./_CPS1/cores
./_CPS1/hbmame
./_CPS1/_Alternatives
./.TemporaryItems
./.TemporaryItems/folders.501
./.TemporaryItems/folders.501/TemporaryItems
./.Trashes
./.Trashes/501
./.Trashes/501/atarirom
./screenshots
./screenshots/Scripts
./games
./games/NES
./games/Spectrum
./games/C64
./games/GAMEBOY
./games/Apple-I
./games/NeoGeo
./games/SHARP MZ SERIES
./games/Amiga
./games/PDP1
./games/AQUARIUS
./games/HT1080Z
./games/MultiComp
./games/TGFX16
./games/Jupiter
./games/TSConf
./games/PET2001
./games/ATARI800
./games/VIC20
./games/Altair8800
./games/ATARI5200
./games/MACPLUS
./games/SPMX
./games/BBCMicro
./games/MSX
./games/Astrocade
./games/AO486
./games/ODYSSEY2
./games/X68000
./games/SMS
./games/AY-3-8500
./games/APOGEE
./games/Amstrad
./games/ORAO
./games/ATARI2600
./games/Coleco
./games/ZX81
./games/Apple-II
./games/MegaCD
./games/SAMCOUPE
./games/TI-99_4A
./games/ARCHIE
./games/QL
./games/VECTOR06
./games/C16
./games/BK0011M
./games/VECTREX
./games/GBA
./games/SNES
./games/Genesis
./saves
./saves/NES
./saves/GAMEBOY
./saves/Genesis
./inputs
./_Arcade
./_Arcade/cores
./_Arcade/mame
./_Arcade/hbmame
./_Arcade/_alternatives
./_Arcade/_alternatives/_Alibaba
./_Arcade/_alternatives/_Amidar
./_Arcade/_alternatives/_Arkanoid
./_Arcade/_alternatives/_Armored Car
./_Arcade/_alternatives/_Asteroid
./_Arcade/_alternatives/_Asteroids Deluxe
./_Arcade/_alternatives/_Bagman
./_Arcade/_alternatives/_Berzerk
./_Arcade/_alternatives/_Bomb Jack
./_Arcade/_alternatives/_Botanic
./_Arcade/_alternatives/_Bubbles
./_Arcade/_alternatives/_Burger Time
./_Arcade/_alternatives/_Burning Rubber
./_Arcade/_alternatives/_Centipede
./_Arcade/_alternatives/_Colony7
./_Arcade/_alternatives/_Crazy Climber
./_Arcade/_alternatives/_Crazy Kong
./_Arcade/_alternatives/_Dig Dug
./_Arcade/_alternatives/_Dig Dug II
./_Arcade/_alternatives/_Disc of tron
./_Arcade/_alternatives/_Donkey Kong
./_Arcade/_alternatives/_Dorodon
./_Arcade/_alternatives/_Eyes
./_Arcade/_alternatives/_Food Fight
./_Arcade/_alternatives/_Frogger
./_Arcade/_alternatives/_Galaga
./_Arcade/_alternatives/_Galaxian
./_Arcade/_alternatives/_Gaplus
./_Arcade/_alternatives/_Green Beret
./_Arcade/_alternatives/_Joust
./_Arcade/_alternatives/_Jump Shot
./_Arcade/_alternatives/_Kick
./_Arcade/_alternatives/_Lady Bug
./_Arcade/_alternatives/_Lost Tomb
./_Arcade/_alternatives/_Lunar Lander
./_Arcade/_alternatives/_Mappy
./_Arcade/_alternatives/_Mayday
./_Arcade/_alternatives/_Moon Patrol
./_Arcade/_alternatives/_Moon War
./_Arcade/_alternatives/_Ms.Pacman
./_Arcade/_alternatives/_Pacman
./_Arcade/_alternatives/_Pacman Plus
./_Arcade/_alternatives/_Pengo
./_Arcade/_alternatives/_Phoenix
./_Arcade/_alternatives/_Pickin
./_Arcade/_alternatives/_Pisces
./_Arcade/_alternatives/_Pleiads
./_Arcade/_alternatives/_Ponpoko
./_Arcade/_alternatives/_Pooyan
./_Arcade/_alternatives/_RallyX
./_Arcade/_alternatives/_Rampage
./_Arcade/_alternatives/_Rescue
./_Arcade/_alternatives/_Robotron
./_Arcade/_alternatives/_Sarge
./_Arcade/_alternatives/_Satan's Hollow
./_Arcade/_alternatives/_Scramble
./_Arcade/_alternatives/_Sinistar
./_Arcade/_alternatives/_Solomons Key
./_Arcade/_alternatives/_Sprint 2
./_Arcade/_alternatives/_Spy Hunter
./_Arcade/_alternatives/_Strategy X
./_Arcade/_alternatives/_Super Bagman
./_Arcade/_alternatives/_Super Breakout
./_Arcade/_alternatives/_Super Cobra
./_Arcade/_alternatives/_Tapper
./_Arcade/_alternatives/_Tetris
./_Arcade/_alternatives/_The End
./_Arcade/_alternatives/_The Tower of Druaga
./_Arcade/_alternatives/_Time Pilot
./_Arcade/_alternatives/_Traverse USA
./_Arcade/_alternatives/_Tron
./_Arcade/_alternatives/_Uniwars
./_Arcade/_alternatives/_Vanvan Car
./_Arcade/_alternatives/_War of the Bugs
./_Arcade/_alternatives/_Xevious
./_Arcade/_alternatives/_Zig Zag
./_Arcade/_Jotego
./_Arcade/_Jotego/_Alternatives
./_Arcade/_Jotego/_Alternatives/_1942
./_Arcade/_Jotego/_Alternatives/_1943
./_Arcade/_Jotego/_Alternatives/_Bionic Commando
./_Arcade/_Jotego/_Alternatives/_Black Tiger
./_Arcade/_Jotego/_Alternatives/_Commando
./_Arcade/_Jotego/_Alternatives/_Double Dragon
./_Arcade/_Jotego/_Alternatives/_Double Dragon II
./_Arcade/_Jotego/_Alternatives/_Ghosts'n Goblins
./_Arcade/_Jotego/_Alternatives/_Gun.Smoke
./_Arcade/_Jotego/_Alternatives/_Popeye
./_Arcade/_Jotego/_Alternatives/_Tiger Road
./_Arcade/_Jotego/_Alternatives/_Vulgus
./_Arcade/_Sega System 1
./_Arcade/_Sega System 1/_Alternatives
./_Arcade/_Sega System 1/_Alternatives/_Flicky
./_Arcade/_Sega System 1/_Alternatives/_I'm Sorry
./_Arcade/_Sega System 1/_Alternatives/_Mister Viking
./_Arcade/_Sega System 1/_Alternatives/_My Hero
./_Arcade/_Sega System 1/_Alternatives/_Regulus
./_Arcade/_Sega System 1/_Alternatives/_Sega Ninja
./_Arcade/_Sega System 1/_Alternatives/_Spatter
./_Arcade/_Sega System 1/_Alternatives/_TeddyBoy Blues
./_Arcade/_Sega System 1/_Alternatives/_Up and Down
./_Arcade/_Sega System 1/_Alternatives/_Wonder Boy
./_Arcade/_Unofficial
./_Other
./_Computer
./_Utility
./_Console
./Filters
./Filters/LCD Effects
./Filters/LCD Effects/Integer Scaling Effects
./Filters/Misc
./Filters/Misc/Horz Interpolation Only
./Filters/Misc/Scanlines (Sharper)
./Filters/Misc/Scanlines (Softer)
./Filters/Normal Upscaling
./Filters/Scanlines (Bright)
./Filters/Scanlines (Bright)/Brightness 110%
./Filters/Scanlines (Bright)/Brightness 115%
./Filters/Scanlines (Bright)/Brightness 120%
./Filters/Scanlines (Bright)/Brightness 125%
./Filters/Scanlines (Bright)/Brightness 130%
./Filters/Scanlines (Vertical)
./Filters/SNES Specific
./Filters/SNES Specific/Misc (Bilinear Sharp)
./Filters/SNES Specific/Misc (Gaussian Sharp)
./Gamma
./Gamma/Poly_Gamma
./Gamma/Pure_Gamma
./cheats
./cheats/NES
./cheats/TGFX16
./cheats/SMS
./cheats/GameBoy
./cheats/Genesis
./cheats/SNES
./cheats/GBA
/media/fat#
```

Please notice that overall MiSTer works OK (for example: Genesis Core and ZX81 Core)

**Comment by sorgelig** [comment]
Please move to forum. There is Arcade topic there. Read it first - you will find many useful info there. If you still have questions - post there.
Github is not a forum for discussions.

**Comment by MacLe7400** [comment]
> Please move to forum. There is Arcade topic there. Read it first - you will find many useful info there. If you still have questions - post there.
> Github is not a forum for discussions.

To be honest... it is open issue for me. I setup my MISTer according to the instructions on wiki and 
arcade cores doesn't work (in my case). So I'm looking for any kind of solution and I'm open on any hints. It is not like open-discussion, just looking for help in my case.

Which forum exactly?

Regards,
M

**Comment by sorgelig** [comment]
http://www.atari-forum.com/viewtopic.php?f=117&t=32481

**Comment by MacLe7400** [comment]
> http://www.atari-forum.com/viewtopic.php?f=117&t=32481

Thank you. I read all topic but I didn't find answer to my issue.

However... I found the way to see the logs (UART connectivity) and isolated the problem eventually. 

I had `/media/fat/mame/`directory (if I remember correctly it was created by script `build_mame_roms.ini` which is no longer needed) which prevented to open mame zip from `/media/fat/_Arcade/mame`).
I removed `/media/fat/mame/` and all cores started to operate correctly.

Maybe it will help someone.

Regads,
M
